<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestLoop</class>
 <widget class="QDialog" name="TestLoop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1677</width>
    <height>1099</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_panel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>时序：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_panel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>501</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_test_count">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>循环数：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_test_count">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>310</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_enter_test">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>560</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_exit_test">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>560</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_machine_back">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>770</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/back.png);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
